<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoMeasure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="SelectBtn">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lblPath">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
          <item>
           <widget class="QGroupBox" name="ImageBox">
            <property name="title">
             <string>图像</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,0,0">
             <item>
              <widget class="CQtOpenCVViewerGl" name="MatDisplay"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QToolButton" name="MatButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="SliderProgress">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="FileButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Frame:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblFrame">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Time:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblMsec">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="TimeBox">
            <property name="title">
             <string>区间</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>FPS:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblFps">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>开始时间</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeStart">
                 <property name="currentSection">
                  <enum>QDateTimeEdit::HourSection</enum>
                 </property>
                 <property name="displayFormat">
                  <string>HH:mm:ss</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>结束时间</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEnd">
                 <property name="currentSection">
                  <enum>QDateTimeEdit::MinuteSection</enum>
                 </property>
                 <property name="displayFormat">
                  <string>HH:mm:ss</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>输出</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="OutputText"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CQtOpenCVViewerGl</class>
   <extends>QOpenGLWidget</extends>
   <header>cqtopencvviewergl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
